MAUI/Scheduler/day-week-views.md

```diff
@@ -590,6 +590,101 @@ N>
 * [View scheduler appointment sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/GettingStarted)
 * [View business object sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/BusinessObject)
 
+## Customize the appearance of more appointments indicator in the all-day panel
```

Comment:
Customize the appearance of more appointments indicator using data template in the all-day panel